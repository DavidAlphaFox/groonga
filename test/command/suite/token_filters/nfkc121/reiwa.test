tokenize \
  'TokenBigram' \
  "ã‹¿" \
  --token_filters 'TokenFilterNFKC121'
