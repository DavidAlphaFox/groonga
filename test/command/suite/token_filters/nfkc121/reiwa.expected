tokenize   'TokenBigram'   "㋿"   --token_filters 'TokenFilterNFKC121'
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "令和",
      "position": 0,
      "force_prefix": true,
      "force_prefix_search": true
    }
  ]
]
